ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ethernet-interfaces.cfg

```diff
@@ -43,9 +75,12 @@ interface Ethernet1
 !
 interface Ethernet2
    description SRV-POD02_Eth1
+   switchport
+   switchport vlan translation in required
+   switchport dot1q vlan tag disallowed
    switchport trunk allowed vlan 110-111,210-211
    switchport mode trunk
-   switchport
+   no switchport
    tcp mss ceiling ipv4 70 ingress
```

Comment:
Not good to have both switchport and no switchport. I hope the suggested changes can fix that.

Reply:
```
s1-leaf1(config-if-Et2)#no switchport 
s1-leaf1(config-if-Et2)#show active
interface Ethernet2
   description P2P_LINK_TO_S1-SPINE1_Ethernet2
   mtu 9000
   no switchport
   ip address 10.1.2.1/31
s1-leaf1(config-if-Et2)#switchport mode trunk ?
  phone  Set trunking mode to phone trunk unconditionally
  <cr>   

s1-leaf1(config-if-Et2)#switchport mode trunk 
s1-leaf1(config-if-Et2)#show active
interface Ethernet2
   description P2P_LINK_TO_S1-SPINE1_Ethernet2
   mtu 9000
   switchport mode trunk
   no switchport
   ip address 10.1.2.1/31
s1-leaf1(config-if-Et2)#
```

Reply:
Fixed